wit/wasi-io.wit

```diff
@@ -0,0 +1,128 @@
+default interface wasi-io {
```

Comment:
Considering that this work is already defined in: https://github.com/WebAssembly/wasi-io/blob/main/wit/wasi-io.wit.md , I probably would avoid duplicating it in this repo. Instead, just refer to it off of a world.